### PR TITLE
Feature/jenkinsfile parallelize

### PR DESCRIPTION
**Pull request by Bitty9**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -16,7 +10,147 @@ pipeline {
...
+                        }
+                        sh '''#!/bin/bash -ex
+                            source uaa-cf-release/config-local/set-env.sh
+                            unset HTTPS_PROXY
+                            unset HTTP_PROXY
+                            unset http_proxy
+                            unset https_proxy
+                            unset GRADLE_OPTS
+                            pushd uaa
+                                apt-get -qy install lsof
```
Comment (6palace) [inline Jenkinsfile]: should we put this in the docker image instead?

File: Jenkinsfile [hunk 2]
```diff
@@ -32,103 +166,18 @@ pipeline {
                     unset http_proxy
                     unset https_proxy
                     unset GRADLE_OPTS
+                    unset DEFAULT_JVM_OPTS
+                    unset JAVA_PROXY_OPTS
+                    unset PROXY_PORT
+                    unset PROXY_HOST
+                    cat /etc/hosts
+                    curl -v http://simplesamlphp2.cfapps.io/saml2/idp/metadata.php
+                    curl -v http://simplesamlphp2.cfapps.io/saml2/idp/metadata.php
```
Comment (6palace) [inline Jenkinsfile]: remove unnecessary curls

File: Jenkinsfile (hunk 1 above)
Comment (bsekar) [reply Jenkinsfile]: Cleaner vs leaner docker image. We can add this to the refactor story

File: Jenkinsfile (hunk 2 above)
Comment (bsekar) [reply Jenkinsfile]: Will be done with the deploy feature

**Pull request merged**